sw/device/lib/dif/dif_plic.c

```diff
@@ -14,7 +14,7 @@
 // If either of these static assertions fail, then the assumptions in this DIF
 // implementation should be revisited. In particular, `plic_target_reg_offsets`
 // may need updating,
-_Static_assert(RV_PLIC_PARAM_NUMSRC == 81,
+_Static_assert(RV_PLIC_PARAM_NUMSRC == 83,
```

Comment:
@lenary @silvestrst that's all that's needed in this case (2 new interrupts), isn't it?

Reply:
In believe so, yes. Could you please run `sw/device/tests/dif/dif_plic_unittest.cc` and see if that doesn't fail?

Reply:
It fails with this PR, the fix is coming once I've confirmed a couple other things.

Reply:
I modified the tests now as well and CI is passing on them, should be OK then.